darwin/DarwinProcess.c

```diff
@@ -248,8 +255,9 @@ void DarwinProcess_setFromLibprocPidinfo(DarwinProcess* proc, DarwinProcessList*
          uint64_t diff = (pti.pti_total_system - proc->stime)
                        + (pti.pti_total_user - proc->utime);
 
-         proc->super.percent_cpu = (double)diff * (double)dpl->super.cpuCount
-                                 / ((double)dpl->global_diff * 100000.0);
+         double time_interval = ticksToNanoseconds(dpl->global_diff) / (double) dpl->super.cpuCount;
+
+         proc->super.percent_cpu = ((double)diff / time_interval) * 100.0;
```

Comment:
Should include some check for `time_interval` to not be too small. Epsilon check for `interval > 1e-6` should be fine.

Reply:
What's the meaning if interval is so small? Two back-to-back updates were done in very short succession?

Reply:
For example. Another possibility is, when the notebook was in standby for a certain time. @cgzones fixed a similar issue for Linux recently, cf. #353 …